src/builder.ts

```diff
@@ -128,6 +128,7 @@ import {
 import { DynamicInputMethodDef, DynamicOutputMethodDef } from "./dynamicMethod";
 import { DynamicOutputPropertyDef } from "./dynamicProperty";
 import { decorateType } from "./definitions/decorateType";
+import { cloneDeep } from "lodash";
```

Comment:
Let's not add lodash as a dependency 😄

Reply:
@tgriesser yes yes! :D I was just debugging something, on a wip that was doing deep clone stuff. All gone now though. 🙏